test/models/bot/alegre_test.rb

```diff
@@ -129,7 +122,7 @@ def setup
     pm2 = create_project_media project: p
     pm3 = create_project_media project: p
     create_relationship source_id: pm3.id, target_id: pm2.id
-    @bot.add_relationships(pm1, [pm2.id])
+    Bot::Alegre.add_relationships(pm1, [pm2.id])
     r = Relationship.last
     assert_equal pm1, r.target
     assert_equal pm3, r.source
```

Comment:
Some tests still need to be implemented... the following lines of `app/models/bot/alegre.rb` are not covered: 7-8, 55-57, 61, 130-131

Reply:
Thanks! Where can I view the uncovered lines?

Reply:
On Code Climate we can see the uncovered files: https://codeclimate.com/repos/58bdc058359261025a0020fa/settings/test_reports/5e575d845d73f11ea80064a0. In order to see the lines, I had to run locally. We need a better way to do this in the parallel Travis build... for example, we can upload the HTML coverage report to S3 as well. There is a Code Climate browser extension that used to show the uncovered lines directly on GitHub, but it's not working at the moment (for me, at least) :(

Reply:
Ideally, we need a way to display the uncovered lines in the build log. It's all about making it easier for devs to find and act on this info.

Reply:
Agree... but we have two parallel jobs. On the Travis matrix there is a way to run a third job after the two parallel ones are done... we can have the third job to only show the uncovered lines.